src/index.js

```diff
@@ -299,6 +299,8 @@ function ncc (
               transpileOnly,
               compiler: eval('__dirname + "/typescript.js"'),
               compilerOptions: {
+                allowSyntheticDefaultImports: true,
```

Comment:
Do we also want to use `esModuleInterop: true`?

Reply:
I'm pretty sure that only affects output (to CJS, which this is now not), not interpretation.